CMS/static/scss/global.scss

```diff
@@ -256,21 +256,19 @@ a.skip-main {
     a {
         font-size: 19px;
         font-family: "HelveticaNeueMedium", Helvetica, Arial, sans-serif;
-        color: #fff;
+        color: #fff !important;
```

Comment:
We would normally try to avoid using !important as it can cause problems down the line. Instead making the css more specific to get the styling to be used.

But given the inherited crc.css in this project I'm happy for this to be used.